crypto/fipsmodule/aes/asm/aesni-x86.pl

```diff
@@ -807,101 +808,6 @@ sub aesni_generate6
 
 	&jmp	(&label("good_key"));
 
-&set_label("12rounds",16);
```

Comment:
Add a comment saying the 192-bit key support was removed here.

Reply:
Resolved [here](https://github.com/briansmith/ring/pull/707/files#diff-15fb5a9556ee9b43d6a1db688b6ac1b8R810)